examples/remix/app/routes/_api/logout.tsx

```diff
@@ -0,0 +1,24 @@
+import { redirect } from "@remix-run/node";
+import type { DataFunctionArgs } from "@remix-run/node";
+import { serialize } from "cookie";
+import { extractHankoCookie } from "~/lib/auth.server";
+import { TodoClient } from "~/lib/todo.server";
+
+// This will first log you out of the express backend and then remove the hanko cookie from
```

Comment:
Since the `todoClient.logout()` function runs on the server, the response it not relayed to the frontend. Hence we have to manually return a response that deletes the cookie.

You can test it by removing `Set-Cookie` header from the `action`.